scripts/docker-machine-port-forwarding.sh

```diff
@@ -1,18 +1,31 @@
 #!/bin/bash
 #
-# Set up kubectl port forwarding to boot2docker VM if needed.
+# Set up kubectl port forwarding to docker-machine VM if needed.
 
+KUBERNETES_API_PORT=8080
+
+function active_machine {
+    if [ $(command -v docker-machine) ]; then
+        docker-machine active
+    fi
+}
+
+# Kube API is accessible on host at 8080, use ssh tunnel
 function forward_port_if_not_forwarded {
-    port=$1
-    machine=$(docker-machine active)
-    forward_port_command="ssh -f -N -i $HOME/.docker/machine/machines/$machine/id_rsa -L $port:localhost:$port docker@$(docker-machine ip $machine)"
-    existing_forward=$(ps ax | grep "$forward_port_command" | grep -v grep)
-    
-    if [ -z "$existing_forward" ]; then
-        eval $forward_port_command
+    local port=$1
+    local machine=$(active_machine)
+
+    if [ -n "$machine" ]; then
+        if ! pgrep -f "ssh.*$port:localhost" > /dev/null; then
+            docker-machine ssh "$machine" -f -N -L "$port:localhost:$port"
+        else
+            echo "An ssh tunnel on port $port already exists.  The kubernetes cluster won't be reachable from local kubectl."
```

Comment:
Can we change the error message to read "Did not set up port forwarding to the Docker machine: An ssh tunnel on port $port already exists. The kubernetes cluster may not be reachable from local kubectl."